plugins/woocommerce-blocks/assets/js/blocks/product-collection/collections/register-product-collection.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import { BlockVariation, registerBlockVariation } from '@wordpress/blocks';
+import { addFilter } from '@wordpress/hooks';
+import { EditorBlock } from '@woocommerce/types';
+import { type ElementType } from '@wordpress/element';
```

Comment:
This looks wrong. It seems like it's importing as both type and value? not sure, I never saw this style before.

I would expect

import type { ElementType }

fwiw if the type can't resolve for some reason you're safe to import it from react instead as long as its only a type import.

Reply:
Good catch! TBH I mostly used auto-import by VS Code. After your comment, I looked into `type` a bit more and learned quite a bit about it—thanks for pointing it out! 🙏